### PR TITLE
Create jursidiction service pod for migrations

### PR DESCRIPTION
**Pull request by drtyh2o**
Creating an `mds-jurisdiction-service` pod. For now, this pod will simply run DB migrations at startup, but eventually will provide a gRPC or other networked interface to the service methods.
Screen Shot 2020-05-01 at 5 13 34 PM

### CODE REVIEW

File: container-images/mds-agency/package.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
   "scripts": {
     "bundle": "webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
     "image": "yarn bundle && ../../bin/mdsctl -c image-name=mds-agency:${npm_package_version} buildImage",
-    "start": "yarn watch server",
```
Comment (macsj200) [inline container-images/mds-agency/package.json:13]: No more `yarn watch server`? Is that being consolidated under a different command, or is this a one-off change for `Agency`?

File: container-images/mds-compliance/package.json [hunk 2]
```diff
@@ -10,7 +10,6 @@
   "scripts": {
     "bundle": "webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
     "image": "yarn bundle && ../../bin/mdsctl -c image-name=mds-compliance:${npm_package_version} buildImage",
-    "start": "yarn watch server",
```
Comment (macsj200) [inline container-images/mds-compliance/package.json:13]: Re. my earlier comment, it looks like we are getting rid of `yarn start` in a few places. I'd like to chat about that more and figure out what the new equivalent is, if any.

File: packages/mds-jurisdiction-service/server/index.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import logger from '@mds-core/mds-logger'
+import { hours } from '@mds-core/mds-utils'
+import { JurisdictionServiceProvider } from '../service/provider'
+
+const signals: NodeJS.Signals[] = ['SIGTERM', 'SIGINT']
```
Comment (macsj200) [inline packages/mds-jurisdiction-service/server/index.ts:21]: I'm not following what this `signals` stuff is intended for, could you elaborate? Thanks!

File: container-images/mds-agency/webpack.config.ts [hunk 4]
```diff
@@ -1,3 +1,3 @@
 import webpack from '@mds-core/mds-webpack-config'
 
-export default webpack.Bundle('../../packages/mds-agency/server.ts').UsingDefaultConfig()
+export default webpack.Bundle('../../packages/mds-agency/server').UsingDefaultConfig()
```
Comment (macsj200) [inline container-images/mds-agency/webpack.config.ts:3]: I know the linter is smart enough to figure this type of thing out in the imports, any chance we could get some rules to apply to this scenario as well?

File: container-images/mds-agency/package.json (hunk 1 above)
Comment (drtyh2o) [reply container-images/mds-agency/package.json:13]: It was only removed from the docker image packages. We don’t use it there. It’s still available when running packages locally.

File: packages/mds-jurisdiction-service/server/index.ts (hunk 3 above)
Comment (drtyh2o) [reply packages/mds-jurisdiction-service/server/index.ts:21]: If you ctrl-c when running locally or kill the k8s pod, this code will listen for the signal and perform a graceful shutdown of the db connections. This is from the logs when a pod is terminated:
```
│ INFO "Initializing Repostory: jurisdictions"                                                                         │
│ INFO "Initializing ro connection: jurisdictions-ro-353d2a18-c14b-4b6b-ba02-f52896e7048c"                             │
│ INFO "Initializing rw connection: jurisdictions-rw-353d2a18-c14b-4b6b-ba02-f52896e7048c"                             │
│ INFO "Ran no migrations (jurisdictions-migrations)"                                                                  │
│ INFO "Running @container-images/mds-jurisdiction-service v0.0.1 (ce943d73) listening for SIGTERM, SIGINT"            │
│ INFO "Shutting down @container-images/mds-jurisdiction-service v0.0.1 (ce943d73) on SIGTERM"                         │
│ INFO "Terminating Repository: jurisdictions"                                                                         │
│ INFO "Terminating ro connection: jurisdictions-ro-353d2a18-c14b-4b6b-ba02-f52896e7048c"                              │
│ INFO "Terminating rw connection: jurisdictions-rw-353d2a18-c14b-4b6b-ba02-f52896e7048c"
```

File: container-images/mds-agency/package.json (hunk 1 above)
Comment (drtyh2o) [reply container-images/mds-agency/package.json:13]: It was only removed from `container-images/mds-agency` not `packages/mds-agency`

File: container-images/mds-compliance/package.json (hunk 2 above)
Comment (drtyh2o) [reply container-images/mds-compliance/package.json:13]: `yarn start` was only removed from `container-images/*` not `packages/*`

File: packages/mds-jurisdiction-service/server/index.ts (hunk 3 above)
Comment (drtyh2o) [reply packages/mds-jurisdiction-service/server/index.ts]: Monitoring for process termination to perform graceful shutdown.

File: container-images/mds-agency/webpack.config.ts (hunk 4 above)
Comment (drtyh2o) [reply container-images/mds-agency/webpack.config.ts:3]: This was only changed to allow loading the entry point from either `server.ts` or `server/index.ts`

**Pull request merged**